src/instamatic/camera/camera_serval.py

```diff
@@ -46,12 +67,32 @@ def get_image(self, exposure=None, binsize=None, **kwargs) -> np.ndarray:
         """
         if exposure is None:
             exposure = self.default_exposure
-        if not binsize:
-            binsize = self.default_binsize
-
+        if exposure < self.MIN_EXPOSURE:
+            logger.warning(f'{self.BAD_EXPOSURE_MSG}: {exposure}')
+            return self._get_image_null()
+        elif self.MIN_EXPOSURE <= exposure <= self.MAX_EXPOSURE:
+            return self._get_image_single(exposure, binsize, **kwargs)
+        else:  # if exposure > self.MAX_EXPOSURE
```

Comment:
Imo, logically _get_image_single should be the default. This also simplifies the logic a bit.

```suggestion
        elif exposure > self.MAX_EXPOSURE:
            ...
        else:
            return self._get_image_single(exposure, binsize, **kwargs)
```